internal/dockerexec/docker.go

```diff
@@ -34,7 +35,9 @@ func New(opts *Options) (*Docker, error) {
 		logger = zap.NewNop()
 	}
 
-	rnd := rand.New(rand.NewSource(uint64(time.Now().UnixNano())))
+	var seed [32]byte
```

Comment:
There is no need to use `math/rand/v2`. `math/rand` will be fine and the code can stay simple:

```go
rnd := rand.New(rand.NewSource(time.Now().UnixNano()))
```